tasks.sh

```diff
@@ -79,7 +79,7 @@ fi
 if [ "${TARGET_ARCH}" = "x86_64" ]; then
   : ${VMLINUX:="bzImage"}
   : ${CLANG_TARGET:="x86_64-linux-gnu"}
-  : ${DEBIAN_TARGET_ARCH:="amd64"}
+  : ${DEBIAN_TARGET_ARCH:="x86-64"}
```

Comment:
I'd call this MKOSI_TARGET_ARCH now since technically we're taking a step away from being too tied to debian. Let's keep debian the default for now but I like that switching to a different guest distro is just one flag away now :)